src/__test__/deserialization.test.ts

```diff
@@ -23,7 +24,13 @@ describe("parseByteModel", () => {
 
 function _parseByteDataToJSON(filename: string) {
   const byte_data = fs.readFileSync(path.resolve(__dirname, filename));
-  return JSON.parse(JSON.stringify(parseFeatherQueryResponse(byte_data)));
+  return JSON.parse(
+    JSON.stringify(
+      parseFeatherQueryResponse(byte_data, {
+        timestampFormat: TimestampFormat.ISO_8601,
```

Comment:
I think it would be nice to add a test case passing `TimeStampFormat.EPOCH_MILLIS` since the table processing function does things in both cases.